simd/unit_tests/include/TestSIMD_MathOps.hpp

```diff
@@ -294,6 +294,9 @@ TEST(simd, host_math_ops) {
 }
 
 TEST(simd, device_math_ops) {
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
+  GTEST_SKIP();
+#endif
```

Comment:
```suggestion
#ifdef KOKKOS_ENABLE_OPENMPTARGET  // FIXME_OPENMPTARGET
  GTEST_SKIP() << "some helpful message";
#endif
```

Reply:
Added a short description about the reported failure. c0c64b1.